hasura/metadata/databases/default/tables/public_redirect.yaml

```diff
@@ -31,12 +31,23 @@ insert_permissions:
   - role: user
     permission:
       check:
-        action:
-          app:
-            team:
-              users:
-                id:
-                  _eq: X-Hasura-User-Id
+        _and:
+          - action:
+              app:
+                team:
+                  memberships:
+                    user_id:
+                      _eq: X-Hasura-User-Id
+          - action:
+              app:
+                team:
+                  memberships:
+                    role:
+                      _neq: MEMBER
```

Comment:
probably more work but best to explicitly allow ADMIN & OWNER vs not allowing MEMBER

Reply:
Done